nginx/config-files/hydroshare-ssl-nginx.conf.template

```diff
@@ -10,16 +15,34 @@ geo $external {
   172.16.0.0/12 0;
   127.0.0.0/8 0;
   10.0.0.0/8 0;
+  152.54.0.0/22 0;
```

Comment:
I missed this in the dev PR but can we make a note to start commenting in sourcecode non obvious IP blocks? The top 3 are standard, but this one is not.